conjure-lib/src/main/java/com/palantir/conjure/java/lib/Bytes.java

```diff
@@ -96,6 +95,7 @@ public static Bytes from(byte[] array, int offset, int length) {
     }
 
     /** Constructs a new {@link Bytes} read from the provided {@link ByteBuffer}. */
+    @JsonCreator
```

Comment:
Would you mind adding a serialized form tests including a negative test for `[]` so it's easier for the next dev to debug this type of failure?